rhtap/download-sbom-from-url-in-attestation.sh

```diff
@@ -120,6 +120,13 @@ fi
 
 jq -r '.components[].containerImage' <<< "$IMAGES" | while read -r image; do
     echo "Getting attestation for $image"
+
+    image_registry="${image/\/*/}"
+    # If the repo is not publicly accessible we need to authenticate so ec can access it
+    prepare-registry-user-pass $image_registry
+    echo "cosign login to registry $image_registry"
+    cosign login --username="$IMAGE_REGISTRY_USER" --password="$IMAGE_REGISTRY_PASSWORD" $image_registry
```

Comment:
Looks like this is happening once per image. Since all the images are likely from the same registry, it would be more efficient to just do it once.

Reply:
(Discussed on slack. Conclusion: Not a big deal, and actually we might in theory have images in different registries.)